files/clappr-rtmp/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "clappr-rtmp-plugin",
+  "repo": "flavioribeiro/clappr-rtmp-plugin",
+  "files": {
+    "basePath": "dist/",
+    "include": ["rtmp.min.js", "rtmp.js", "./assets/**/*"]
```

Comment:
the problem is that the plugin build generates not only these files but more on dist/. This include will filter only the necessary ones in case the others are committed/pushed.